static/js/components/dashboard/courses/StatusMessages.js

```diff
@@ -123,7 +123,7 @@ const messageForNotAttemptedEdxExam = (course: Course) => {
         {
           "You are authorized to take the virtual proctored exam for this course. Please "
         }
-        <a href="https://edx.org">
+        <a href={course.exam_url}>
```

Comment:
Should we do the same check for `course.exam_url` in the if statement like we're doing below?